cmd/zc_traverser_file.go

```diff
@@ -118,12 +119,17 @@ func (t *fileTraverser) traverse(processor objectProcessor, filters []objectFilt
 				relativePath = strings.TrimPrefix(relativePath, common.AZCOPY_PATH_SEPARATOR_STRING)
 
 				// We need to omit some properties if we don't get properties
+				// TODO: make it so we can (and must) call newStoredOBject here.
+				//     Reviewers: don't pass this if I have not fixed this when you review!!!
```

Comment:
On second thoughts. Let's not worry about calling newStoredObject in THIS PR.  If/when we fix that, let's do it in a different PR